modules/core/src/smithy4s/http/Metadata.scala

```diff
@@ -267,16 +251,6 @@ object Metadata {
   ) extends CachedSchemaCompiler.DerivingImpl[Encoder]
       with EncoderCompiler {
 
-    @deprecated
```

Comment:
Just noting this didn't have a message or version in case that warrants scrutiny.